src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -437,13 +438,63 @@ object Helpers extends SparkSessionWrapper {
     s"SHRED COMPLETE: ${target.tableFullName}"
   }
 
+
+  /**
+   * Execute a parallelized streaming to follow the instructions provided through CloneDetail class
+   *
+   * @param cloneDetails details required to execute the parallelized clone
+   * @return
+   */
+  private[overwatch] def snapStream(cloneDetails: Seq[CloneDetail],snapshotRootPath: String): Unit = {
+
+    val cloneDetailsPar = cloneDetails.par
+    val taskSupport = new ForkJoinTaskSupport(new ForkJoinPool(parallelism))
+    cloneDetailsPar.tasksupport = taskSupport
+
+    logger.log(Level.INFO, "Streaming START:")
+    val cloneReport = cloneDetailsPar.map(cloneSpec => {
+      try {
+        val rawStreamingDF = spark.readStream.format("delta").load(s"${cloneSpec.source}")
+        val sourceName = s"${cloneSpec.source}".split("/").takeRight(1).head
+        val checkPointLocation = if (snapshotRootPath.takeRight(1) == "/") s"${snapshotRootPath}checkpoint/${sourceName}" else s"${snapshotRootPath}/checkpoint/${sourceName}"
+        val cloneReportPath = if (snapshotRootPath.takeRight(1) == "/") s"${snapshotRootPath}report/" else s"${snapshotRootPath}/clone_report/"
```

Comment:
use the new path slash helpers introduced in https://github.com/databrickslabs/overwatch/pull/825 -- these are merged to 0720 now so rebase and you should have access to them.